composer.json

```diff
@@ -18,7 +18,8 @@
         }
     ],
     "require": {
-        "semsol/arc2": "v2.2.4"
+        "semsol/arc2": "v2.2.4",
+        "chrisboulton/php-resque": "dev-master#98fde571db008a8b48e73022599d1d1c07d4a7b5"
```

Comment:
1.2 is nearly 3 years old and the php-resque project party line is to use dev-master.  See: https://github.com/chrisboulton/php-resque/issues/203#issuecomment-88145890